linera-views/tests/random_container_tests.rs

```diff
@@ -370,10 +370,14 @@ async fn run_map_view_mutability<R: RngCore + Clone>(rng: &mut R) -> Result<()>
                 let part_key_values = view.map.key_values_by_prefix(vec![u]).await?;
                 assert_eq!(part_state_vec, part_key_values);
             }
-            for key in &all_keys {
+            let keys_vec = all_keys.clone().into_iter().collect::<Vec<_>>();
+            let values = view.map.multi_get(keys_vec.clone()).await?;
+            for i in 0..keys_vec.len() {
+                let key = &keys_vec[i];
                 let test_map = new_state_map.contains_key(key);
                 let test_view = view.map.get(key).await?.is_some();
```

Comment:
Maybe `map_has_key` and `view_has_key`?

Reply:
I added the use of `contains_key` next to `get`. This is test code after all.